app/assets/stylesheets/catarse_bootstrap/catarse.css.sass

```diff
@@ -3,6 +3,8 @@
 @import catarse_bootstrap/normalize
 
 // The order below DOES matter. Extra should have priority.
+@import catarse_bootstrap/custom_mixins
```

Comment:
Anything not in the main should go to the extra file, so we can better isolate the CSS generated by hand (not from webflow)